tests/HomeworkCalculatorTesting.spec.js

```diff
@@ -0,0 +1,82 @@
+const { test, expect } = require('@playwright/test');
+/*const { BasicCalculatorPage } = require('../pages-Calculator/basicCalculatorPage');
+
+
+test.describe('', () => {
+    let page;
+    
+    test.beforeAll(async ({ browser }) => {
+      page = await browser.newPage();
+      startPage = new BasicCalculatorPage(page);
+
+        });
+   test.beforeEach(async () => {
+      await startPage.goto();
+    });
+  });*/
+       
+ 
+test('Testing page to be opened', async ({ page }) => {
+  await page.goto('https://testsheepnz.github.io/BasicCalculator');
+  const calculatorPageTop = await page.isVisible('#page-top');
+  expect(calculatorPageTop).toBe(true);
```

Comment:
variable name should be more clear. in this case isCalculatorPageTopVisible could be used.